ui/packages/tidb-dashboard-for-debugportal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pingcap/tidb-dashboard-for-debugportal",
-  "version": "0.0.2",
+  "version": "0.0.3",
```

Comment:
0.0.3 and 0.0.4 has been published in another PR: https://github.com/pingcap/tidb-dashboard/pull/1390/files#diff-db0c94f8dee3a3b841754804e76819d9fc503a4ed82e88972f6c5616d52b9c5d

Reply:
Get!